src/ui/login/LoginFormContainer.js

```diff
@@ -15,6 +16,7 @@ export const mapStateToProps = state => ({
 // map the props
 export const mapDispatchToProps = dispatch => ({
```

Comment:
just FYI (I'm not requiring this change): this could be rewritten as:
```javascript
export const mapDispatchToProps = {
    performLogin,
    setLanguage: setCurrentLanguage,
};
```